jwst/jump/jump.py

```diff
@@ -66,96 +35,22 @@ def detect_jumps(input_model, gain_model, readnoise_model,
-                          frames_per_group, flag_4_neighbors,
-                          max_jump_to_flag_neighbors, min_jump_to_flag_neighbors))
-    # last slice get the rest
-    slices.insert(numslices - 1, (data[:, :, (numslices - 1) * yincrement:nrows, :],
-                                  gdq[:, :, (numslices - 1) * yincrement:nrows, :],
-                                  readnoise_2d[(numslices - 1) * yincrement:nrows, :],
-                                  rejection_thresh, three_grp_thresh, four_grp_thresh,
-                                  frames_per_group, flag_4_neighbors,
-                                  max_jump_to_flag_neighbors, min_jump_to_flag_neighbors))
-    if numslices == 1:
-        gdq, row_below_dq, row_above_dq = twopt.find_crs(data, gdq, readnoise_2d, rejection_thresh,
-                                                         three_grp_thresh, four_grp_thresh,
-                                                         frames_per_group, flag_4_neighbors,
-                                                         max_jump_to_flag_neighbors,
-                                                         min_jump_to_flag_neighbors)
-        elapsed = time.time() - start
-    else:
-        pool = multiprocessing.Pool(processes=numslices)
-        # Starts each slice in it's own process. Starmap allows more than one parameter to be passed.
-        real_result = pool.starmap(twopt.find_crs, slices)
-        pool.close()
-        pool.join()
-        k = 0
-        # Reconstruct gdq, the row_above_gdq, and the row_below_gdq from the slice result
-        for resultslice in real_result:
+        readnoise_2d = reffile_utils.get_subarray_data(input_model,
+                                                       readnoise_model)
 
-            if len(real_result) == k + 1:  # last result
-                gdq[:, :, k * yincrement:nrows, :] = resultslice[0]
-            else:
-                gdq[:, :, k * yincrement:(k + 1) * yincrement, :] = resultslice[0]
-            row_below_gdq[:, :, :] = resultslice[1]
-            row_above_gdq[:, :, :] = resultslice[2]
-            if k != 0:  # for all but the first slice, flag any CR neighbors in the top row of the previous slice and
-                # flag any neighbors in the bottom row of this slice saved from the top of the previous slice
-                gdq[:, :, k * yincrement - 1, :] = np.bitwise_or(gdq[:, :, k * yincrement - 1, :],
-                                                                 row_below_gdq[:, :, :])
-                gdq[:, :, k * yincrement, :] = np.bitwise_or(gdq[:, :, k * yincrement, :],
-                                                             previous_row_above_gdq[:, :, :])
-            # save the neighbors to be flagged that will be in the next slice
-            previous_row_above_gdq = row_above_gdq.copy()
-            k += 1
-        elapsed = time.time() - start
+    # turn off MP for now
+    # max_cores = None
```

Comment:
Does this mean `jump` multiprocessing doesn't work right now?  Is there an existing test for this that's commented out?  Or no test?

Reply:
Multiprocessing is not enabled right now. There are some tests - `test_flagging_of_CRs_across_slice_boundaries` for example. Should I comment them out?

Reply:
Yes, if MP is not possible to run, then the tests that use it need to be skipped. Can use the "@pytest.mark.skip()" directive, rather than commenting them out.